js/main.js

```diff
@@ -14,51 +12,81 @@ function storageAvailable(type) {
   } catch (e) {
     return e instanceof DOMException && (
       e.code === 22
-      || e.code === 1014
-      || e.name === 'QuotaExceededError'
-      || e.name === 'NS_ERROR_DOM_QUOTA_REACHED')
+        || e.code === 1014
+        || e.name === 'QuotaExceededError'
+        || e.name === 'NS_ERROR_DOM_QUOTA_REACHED')
       && (storage && storage.length !== 0);
   }
 }
 
-function updateLocalStorage() {
-  // Updates the Local Storage
-  if (storageAvailable('localStorage')) {
-    localStorage.setItem('books', JSON.stringify(books));
-  }
-}
+class Book {
+  // Book Class has a title and author
+  title;
 
-const localBooksData = localStorage.getItem('books');
+  author;
 
-// Load initially stored data
-if (localBooksData) {
-  books = JSON.parse(localBooksData);
+  constructor(title, author) {
+    this.title = title;
+    this.author = author;
+  }
 }
 
-// Books Related Values & Functions
-const booksList = document.getElementById('books-list');
+class Library {
+  // Library class consists of a list of books
+  books = [];
```

Comment:
- [ ] Good job refactoring this code, but you miss a constructor method in your class Kindly add a constructor method to your `Library` class as follows: 

```JS
class Books {
  constructor() {
    this.books = [];
  }
}
```

Reply:
I have fixed this.